src/Components/Book.js

```diff
@@ -2,16 +2,53 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { deleteBook } from '../redux/Books/booksSlice';
+import '../Style/books.css';
 
 const Book = ({ book }) => {
   const { title, author } = book;
   const dispatch = useDispatch();
   return (
-    <ul>
-      <li>{title}</li>
-      <li>{author}</li>
-      <li><button type="submit" onClick={() => dispatch(deleteBook(book))}>Delete</button></li>
-    </ul>
+    <div className="books-card">
+      <div className="card-left">
+        <h5>Action</h5>
+        <h1 className="books_title">{title}</h1>
+        <p className="books_author">{author}</p>
+        <ul className="action">
+          <li>
+            <button type="button">Comments</button>
+          </li>
+          <div className="seperator" />
+          <li>
+            <button type="submit" onClick={() => dispatch(deleteBook(book))}>
+              Delete
+            </button>
+          </li>
+          <div className="seperator" />
+          <li>
+            <button type="button">Edit</button>
+          </li>
+        </ul>
```

Comment:
- [ ] There's vertical between eact of these buttons and they
```
   background-color:  transparent;
   border: none
   outline: none
   color: `same color on the template`
```
Kindly fix those areas.

### Your Design
![your-btn-actions](https://github.com/KAYEMBAJONATHAN/Book_s/assets/26597432/4800c552-cc79-43ec-b797-4a8a0f59e19e)

### Guiding Template
![temp-btn-actions](https://github.com/KAYEMBAJONATHAN/Book_s/assets/26597432/48995782-fcef-4ebf-9a09-7e79589cf40a)